### PR TITLE
Move css rules from inPlaceReplace.ts into inPlaceReplace.css

### PR DESCRIPTION
**Pull request by Kalmaegi**
For #165169

- [x] src/vs/editor/contrib/inPlaceReplace/browser/inPlaceReplace.ts
- [x]  src/vs/editor/contrib/inlineCompletions/browser/ghostTextWidget.ts
- [x]  src/vs/editor/contrib/links/browser/links.ts
x  src/vs/editor/contrib/find/browser/findOptionsWidget.ts

### CODE REVIEW

**Comment by weartist** [comment]
@microsoft-github-policy-service agree

File: src/vs/editor/contrib/find/browser/findOptionsWidget.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+.monaco-editor .findOptionsWidget {
+	color: var(--vscode-editorWidget-foreground);
+	background-color: var(--vscode-editorWidget-background);
+	box-shadow: var(--vscode-widget-shadow);
```
Comment (aeschli) [inline src/vs/editor/contrib/find/browser/findOptionsWidget.css:10]: That should be `0 0 8px 2px var(--vscode-widget-shadow)`

File: src/vs/editor/contrib/find/browser/findOptionsWidget.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	border: var(--vscode-contrastBorder);
```
Comment (aeschli) [inline src/vs/editor/contrib/find/browser/findOptionsWidget.css:11]: Also here, `2px solid` is missing

**Review by aeschli — CHANGES_REQUESTED** [review]

File: src/vs/editor/contrib/find/browser/findOptionsWidget.css (hunk 1 above)
Comment (weartist) [reply src/vs/editor/contrib/find/browser/findOptionsWidget.css:10]: oh! Thanks😆

**Review by aeschli — DISMISSED** [review]

**Review by dbaeumer — DISMISSED** [review]

**Comment by aeschli** [comment]
Thanks @weartist !

**Comment by weartist** [comment]
@aeschli Looks like it needs to be reviewed, thank you!

**Comment by weartist** [comment]
@aeschli Looks like it needs to be reviewed, thank you! I'll go and try to fix the rest

**Comment by weartist** [comment]
@aeschli I fixed the code conflict. hope it was right

**Comment by weartist** [comment]
@aeschli the findOptionsWidget has been merged with other people's changes, i remove commit of this file

**Review by dbaeumer — DISMISSED** [review]

File: src/vs/editor/contrib/inlineCompletions/browser/ghostText.css [hunk 3]
```diff
@@ -36,3 +36,11 @@
 	text-decoration: underline;
 	text-underline-position: under;
 }
+
+.monaco-editor .ghost-text-decoration,
+.monaco-editor .ghost-text-decoration-preview,
+.monaco-editor .suggest-preview-text .ghost-text {
+	color: var(--vscode-editorGhostText-foreground);
```
Comment (aeschli) [inline src/vs/editor/contrib/inlineCompletions/browser/ghostText.css:43]: `!important` should be on `color`, not on `backgound-color`

**Review by aeschli — CHANGES_REQUESTED** [review]

**Comment by weartist** [comment]
@aeschli  I moved the 'links.css color' position, there was a small mistake  in a previous commit https://github.com/microsoft/vscode/pull/166258/files/63816e279be3c11d4a44ac95721dfb31bb6230ae

**Review by aeschli — APPROVED** [review]

**Review by dbaeumer — APPROVED** [review]

**Pull request merged**